src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml

```diff
@@ -49,6 +49,7 @@
                         Grid.Column="0"
                         Clicked="IsAddingStartingLocations_Clicked"
                         IsEnabled="False"
+                        MinimumHeightRequest="75"
```

Comment:
I suggest setting the grids row heights to "auto" to resolve the cause of the issue.